odk2-src/app-designer-prereqs.rst

```diff
@@ -90,6 +90,11 @@ After installing NodeJS, open a :program:`terminal` (which you can do by clickin
 
   outputs a message telling you permission is denied, then you will have to change the ownership of the :file:`/usr/local/` and :file:`/usr/local/bin/` directories. On Mac, follow the `instructions to take ownership <http://osxdaily.com/2013/04/23/change-file-ownership-mac-os-x/>`_ of these directories, or to at least give yourself read permission. On other Unix systems, use the :program:`chown` command or the user-interface appropriate to your distribution to do so.
 
+
+.. warning::
+
+  Add the location of the :program:`npm` folder to the *PATH* variable of your system if subsequent calls to access grunt in the next step fail. Ex. :file:`C:\\Users\\[username]\\AppData\\Roaming\\npm`
```

Comment:
I think this should be clearer that its a Windows tip (separate from the Mac/Unix tips above). I think you should create a new section "For Windows" that includes this tip.